lib/config.js

```diff
@@ -68,7 +68,15 @@ module.exports = class Config {
         watchOptions: Joi.object().default().keys({
           ignored: Joi.array().default('node_modules')
         }),
-        server: Joi.object().default({}),
+        server: Joi.default({})
```

Comment:
How come `object` came out here?

Reply:
I guess because it can be `false` now?

Reply:
Yeah, we need to default to empty object before we check `Joi.exist('proxy')`, otherwise that check passess, the server is set to `{}` afterwards and then browser-sync will fail the proxy option (`Server must be false with proxy option…`)

Reply:
👍